_redirects

```diff
@@ -437,6 +437,18 @@ layout: null
 /docs/releases/cockroachcloud-12062021.html /docs/releases/cloud.html 301
 /docs/releases/cockroachcloud-12112020.html /docs/releases/cloud.html 301
 
+#CockroachDB Self-Hosted release notes
+
+{% assign rn_rdr = "v20.2,v21.1,v21.2" | split: "," -%}
```

Comment:
Nice!

Reply:
Thanks! This made redirects much easier. And I tested a few of them and they seem to redirect just fine.